ql/src/test/queries/clientpositive/distinct_groupby.q

```diff
@@ -0,0 +1,57 @@
+--! qt:dataset:src1
+
```

Comment:
could we have some tests for non-cbo path as well?

Reply:
adding q tests for non-cbo as well, good idea!